js/functions-and-styling/1-love-calculator.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var firstName = 'Dan', secondName = 'Kabe';
+var calcLove = function(name1, name2){
+
+    var nameLength = name1.length+name2.length-(Math.round(Math.random()*7));
+
+    var result = nameLength*42;
+
+    if(result >100) {
```

Comment:
This can be replaced by a single ternary operator. 
    return (result > 100) ? 100 : result;